scripts/release/publish-build-artifacts.sh

```diff
@@ -8,48 +8,67 @@ set -e -o pipefail
 # Go to the project root directory
 cd $(dirname $0)/../..
 
-buildDir="dist/releases/material"
-buildVersion=$(sed -nE 's/^\s*"version": "(.*?)",$/\1/p' package.json)
+# Material packages that need to published.
+PACKAGES=(cdk material)
+REPOSITORIES=(cdk-builds material2-builds)
 
-commitSha=$(git rev-parse --short HEAD)
-commitAuthorName=$(git --no-pager show -s --format='%an' HEAD)
-commitAuthorEmail=$(git --no-pager show -s --format='%ae' HEAD)
-commitMessage=$(git log --oneline -n 1)
+# Function to publish artifacts of a package to Github.
+#   @param $1 Name of the package
+#   @param $2 Repository name of the package.
+publishPackage() {
 
-repoName="material2-builds"
-repoUrl="https://github.com/angular/material2-builds.git"
-repoDir="tmp/$repoName"
+  buildDir="dist/releases/$1"
```

Comment:
Can you add 
```sh
packageName=${1}
publishRepoName=${2}
```
And then use those variables for the rest of the function?

Also, FYI, it's generally considered good practice to always use curly braces when referencing bash vars:
http://stackoverflow.com/questions/8748831/when-do-we-need-curly-braces-in-variables-using-bash